File: app/src/main/java/moe/yuuta/mipushtester/MainActivity.java
package moe.yuuta.mipushtester;

import android.os.Bundle;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;
import androidx.navigation.Navigation;
import androidx.navigation.ui.NavigationUI;

public class MainActivity extends AppCompatActivity {
    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        NavigationUI.setupActionBarWithNavController(this, Navigation.findNavController(this, R.id.nav_host));
    }

    @Override
    public boolean onSupportNavigateUp() {
        return Navigation.findNavController(this, R.id.nav_host).navigateUp();
    }
}

File: app/src/main/java/moe/yuuta/mipushtester/MainActivity.kt
package moe.yuuta.mipushtester

import android.os.Bundle

import androidx.annotation.Nullable
import androidx.appcompat.app.AppCompatActivity
import androidx.navigation.Navigation
import androidx.navigation.ui.NavigationUI

class MainActivity : AppCompatActivity() {
    @Override
    override fun onCreate(@Nullable savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        NavigationUI.setupActionBarWithNavController(this, Navigation.findNavController(this, R.id.nav_host))
    }

    @Override
    override fun onSupportNavigateUp(): Boolean =
        Navigation.findNavController(this, R.id.nav_host).navigateUp()
}
